### PR TITLE
Add PID example

### PR DESCRIPTION
**Pull request by danielfett**
PID example in line with our current thinking about PID design.

### CODE REVIEW

**Comment by bc-pi** [comment]
https://drafts.oauth.net/oauth-sd-jwt-vc/danielfett/pid-example/draft-ietf-oauth-sd-jwt-vc.html#appendix-B for folks playing along at home

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 1]
```diff
@@ -714,6 +721,51 @@ by [@!RFC5785].
 * Specification document: [[ (#jwt-vc-issuer-metadata) of this of this specification ]]
 * Related information: (none)
 
+# Examples
+
+Important: The following examples are not normative and provided for
+illustration purposes only. In particular, neither the structure of the claims
```
Comment (bc-pi) [inline draft-ietf-oauth-sd-jwt-vc.md:727]:
```suggestion
illustrative purposes only. In particular, neither the structure of the claims
```

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 2]
```diff
@@ -714,6 +721,51 @@ by [@!RFC5785].
...
+
+The following payload is used for the SD-JWT:
+
+<{{examples/03-pid/sd_jwt_payload.json}}
+
+The following Disclosures are created by the Issuer:
+
+{{examples/03-pid/disclosures.md}}
+
+The following is how a presentation of the SD-JWT with a Key Binding JWT that discloses only nationality and the fact that the person is over 18 years old could look like:
```
Comment (bc-pi) [inline draft-ietf-oauth-sd-jwt-vc.md:757]: "could look like" has never sounded right to me

```suggestion
The following is how a presentation of the SD-JWT with a Key Binding JWT that discloses only nationality and the fact that the person is over 18 years old might look:
```

**Comment by bc-pi** [comment]
can haz document history entry?

**Review by awoie — APPROVED** [review]

**Review by bc-pi — APPROVED** [review]

**Pull request merged**